panel/src/components/Forms/Fieldset.vue

```diff
@@ -47,10 +47,23 @@
 </template>
 
 <script>
+import { computed } from "vue";
+
 /**
  * The Fieldset component is a wrapper around manual field component creation. You simply pass it an fields object and all field components will automatically be created including a nice field grid. This is the ideal starting point if you want an easy way to create fields without having to deal with a full form element.
  */
 export default {
+	provide() {
+		return {
+			context: computed(() => this.context)
+		};
+	},
+	inject: {
+		injectedContext: {
```

Comment:
I think you need to explain this to me, I don't quite understand what's happening here

Reply:
each fieldset provides a "context" (which combines the fields it received with its own fields) to child fieldsets. 
This is done here:
```js
provide() {
	return {
                // wrap in `computed` to make it a reactive variable
		context: computed(() => this.context)
	};
},
```
`this.context` refers to the computed value:
```js
computed: {
	context() {
		return {
			...this.injectedContext,
			...this.value
		};
	}
},
```

---- 

While injecting the context from parent fieldsets, we **rename** that provided property to `injectedContext`. It could alse be `parentContext` if that makes it clearer. 
It's just to distinguish the injected (parent) context from the locally computed context (otherwise the computed method would infinitely call itself).